README.md

```diff
@@ -220,9 +222,9 @@ Finally create a [Facelets][20] file `/hello.xhtml` as below:
 
 Start the server and open it by `http://localhost:8080/contextname/hello.xhtml`.
 
-## Activating CDI in Jakarta Faces 3.0
+## Activating CDI in Jakarta Faces 4.0
 
-By default, Jakarta Faces 3.0 will run in Jakarta Faces 2.2 modus as to CDI support. Even when you use a Jakarta Faces 2.3 or 3.0 compatible `faces-config.xml`. In other words, the Jakarta Faces 2.3+ feature of injection and EL resolving of Jakarta Faces artifacts ([spec issue 1316](https://github.com/javaee/javaserverfaces-spec/issues/1316)) won't work until you explicitly activate this. In other words, `@Inject FacesContext` doesn't work by default. This was necessary in order for Jakarta Faces 2.3 to stay compatible with 2.2, and this carried over to 3.0.
+By default, Jakarta Faces 4.0 will run in Jakarta Faces 2.2 modus as to CDI support. Even when you use a Jakarta Faces 2.3 or 3.0 compatible `faces-config.xml`. In other words, the Jakarta Faces 2.3+ feature of injection and EL resolving of Jakarta Faces artifacts ([spec issue 1316](https://github.com/javaee/javaserverfaces-spec/issues/1316)) won't work until you explicitly activate this. In other words, `@Inject FacesContext` doesn't work by default. This was necessary in order for Jakarta Faces 2.3 to stay compatible with 2.2, and this carried over to 3.0.
```

Comment:
Is this still right for JSF 4.0?

Reply:
Nope, Faces 4.0 will definitely not have a fake Faces 2.2 mode ;) It wil default to just Faces 4.0.

Reply:
Thanks for your reply. Just updated this. Feel free to further modify this description. :-)

Day-job-question: Some quick answer how this works together with https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/jsf/el/SpringBeanFacesELResolver.html?
(We already added Weld a few years ago to fulfil CDI-requirement for Omnifaces.)